Add Classifier Proccessor issue/#16

smnv-as opened the pull request:
Changes to be committed:
	modified:   import_me/processors.py
	modified:   tests/test_processors.py

smnv-as committed:
Add Classifier Proccessor issue/#16

Changes to be committed:
	modified:   import_me/processors.py
	modified:   tests/test_processors.py

yakovistomin commented on tests/test_processors.py:
```diff
@@ -498,3 +498,136 @@ def test_choice_processor_exception(value, choices, raw_value_processor, expecte
     assert exc_info.value.messages == [expected_error_message]
+
+
+choices_classifier_no_processor = [
```
Может вынести это и остальные аналогичные переменные в conftest.py?

smnv-as replied:
Просто как глобальные переменные? Просто пока не много не могу понять назначение данного файла.

smnv-as replied:
Вынес как глобальные переменные.

Melevir commented:
@Valerich смержи этот пр сам пжл после того, как тебе будет ок

smnv-as committed:
Refactoring issue/#16

Changes to be committed:
	modified:   tests/conftest.py
	modified:   tests/test_processors.py

Melevir commented:
👍
